substrate/frame/asset-conversion/src/lib.rs

```diff
@@ -685,16 +673,16 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::swap_tokens_for_exact_tokens())]
 		pub fn swap_tokens_for_exact_tokens(
 			origin: OriginFor<T>,
-			path: BoundedVec<T::MultiAssetId, T::MaxSwapPathLength>,
-			amount_out: T::AssetBalance,
-			amount_in_max: T::AssetBalance,
+			path: Vec<Box<T::MultiAssetId>>,
```

Comment:
How is an extrinsic with this in bounded? I know it was suggested that BoundedVec made no sense for runtime api / rpc calls but for extrinsics that could have to be sent via xcm to other chains isn't bounding still important? At least at the moment it would not be safe to make this call over xcm if we structure it like this until the safe call is no longer needed.

Reply:
yeah, if we need them whitelisted for xcm transact, we should have them bounded temporarily

Reply:
we should be good now, since we starting to use the message queue 
https://github.com/paritytech/polkadot-sdk/pull/1246